test/fixtures.ts

```diff
@@ -119,9 +111,9 @@ export let COMBINED_REFINEMENTS = [
 ];
 
 export let CUSTOM_PARAMS_FROM_STRING = [
```

Comment:
Not used anywhere, should we still have tests in query for custom url params now that its moved to bridge or take this out?